addon/content/components/compose/composeWidget.jsx

```diff
@@ -99,11 +100,11 @@ export function ComposeWidget({ discard }) {
       />
       <div id="sendStatus">{composeState.sendingMsg}</div>
       <div className="buttons">
-        {discard && (
-          <a className="link" onClick={discard}>
-            {browser.i18n.getMessage("compose.discard")}
-          </a>
-        )}
+        <button id="discard" onClick={discard} disabled={!discard}>
+          <SvgIcon ariaHidden={true} hash="delete_forever" />
+          &nbsp;
+          {browser.i18n.getMessage("compose.discard")}
```

Comment:
For consistency with the rest of the UI, let's make both of these buttons have the icon first, and then the text.

If you drop the `&nbsp;` you can use `margin-inline-end: 0.1em;` within a `.compose > .buttons > button > .icon` to apply a bit of spacing.

Reply:
Ok